internal/auth/issuer/pam/oidc_issuer.go

```diff
@@ -0,0 +1,45 @@
+package pam
```

Comment:
a little echoing in the namespace would something like below be more clear?

```
internal/auth/oidc/pam/provider.go
```

Reply:
will move this on the next PR if possible